service/src/main/java/bio/terra/workspace/service/resource/controlled/cloud/gcp/gcsbucket/UpdateGcsBucketStep.java

```diff
@@ -108,6 +113,13 @@ private StepResult updateBucket(@Nullable ApiGcpGcsBucketUpdateParameters update
       bucketCowBuilder.setStorageClass(gcsStorageClass);
     }
 
+    final boolean replaceCloningInstructions = null != updateParameters.getCloningInstructions();
```

Comment:
I think this is in the wrong place. I think updating the cloning instructions should follow the update metadata path and not be part of this step. Also, I think it can be done in the same update transaction as the other DAO update. We do not need to run a separate transaction.

Reply:
Good point.